google/tests/playsubstatus/playsubstatus.test.ts

```
@@ -1,15 +1,17 @@
 import { handler } from "../../src/playsubstatus/playsubstatus";
 
 test("Parse and return HTTP 200", () => {
-    /*let handlerResult = handler({
-        headers: {
-            "Play-Purchase-Token": "hoofgeonpgdfcnjdiofeadbe.AO-J1Oz1chlwrlUQnFQDBD2J3JDFiVPXnvkG7BYSYrv9FsjdWf4yyJd76jDhorOcTUU-Zx3r2VYccNAlFGljcdvPnsglwXWivMrR9b6rE7OIg4WGdLCR1W9bFu_lQvltMnQUQlGgkZ4WYe_eTt44wml5TatT70ew8g"
-        },
-        pathParameters: {
-            "subscriptionId": "com.guardian.subscription.monthly.10"
-        }
-    });
-
-    handlerResult.then(result => expect(result.statusCode).toBe(200))*/
+    // process.env['Stage'] = "CODE";
+    // expect.assertions(1);
+    // let handlerResult = handler({
+    //     headers: {
+    //         "Play-Purchase-Token": "hoofgeonpgdfcnjdiofeadbe.AO-J1Oz1chlwrlUQnFQDBD2J3JDFiVPXnvkG7BYSYrv9FsjdWf4yyJd76jDhorOcTUU-Zx3r2VYccNAlFGljcdvPnsglwXWivMrR9b6rE7OIg4WGdLCR1W9bFu_lQvltMnQUQlGgkZ4WYe_eTt44wml5TatT70ew8g"
+    //     },
+    //     pathParameters: {
+    //         "subscriptionId": "com.guardian.subscription.monthly.10"
+    //     }
+    // });
+    //
+    // return handlerResult.then(result => expect(result.statusCode).toBe(200));
```

This test won't fail silently anymore (fixed as per https://github.com/guardian/mobile-purchases/pull/45), but I've commented it out again for now because it won't work when run via TeamCity